### PR TITLE
Update link for Galaxy configuration file in Galaxy debug document

### PR DESCRIPTION
**Pull request by kxk302**

### CODE REVIEW

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 1]
```diff
@@ -0,0 +1,1434 @@
+#
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]: Can you remove all the commented out stuff? If you're using this you hopefully know what options to set.

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 2]
```diff
@@ -0,0 +1,1434 @@
...
+
+# Run the Python profiler on each request.
+#use_profile = False
+
+# Intercept print statements and show them on the returned page.
+#use_printdebug = True
+
+# Enable live debugging in your browser.  This should NEVER be enabled on a
+# public site.  Enabled in the sample config for development.
+use_interactive = True
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]:
```suggestion
# use_interactive = True
```
That doesn't work anymore.

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 3]
```diff
@@ -0,0 +1,1434 @@
...
+# where a malicious party could provide a link that appears to reference the
+# Galaxy server, but contains a redirect to a third-party server, tricking a
+# Galaxy user to access said site.
+#enable_old_display_applications = True
+
+# -- Next gen LIMS interface on top of existing Galaxy Sample/Request
+# management code.
+
+use_nglims = False
+nglims_config_file = tool-data/nglims.yaml
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]:
```suggestion
# use_nglims = False
# nglims_config_file = tool-data/nglims.yaml
```

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 4]
```diff
@@ -0,0 +1,1434 @@
...
+# can cause performance problems, so using job_working_directory ('.' or cwd
+# when a job is run) is encouraged.  By default, both are checked to avoid
+# breaking existing tools.
+#collect_outputs_from = new_file_path,job_working_directory
+
+# -- Data Storage (Object Store)
+#
+# Configuration file for the object store
+# If this is set and exists, it overrides any other objectstore settings.
+object_store_config_file = config/object_store_conf.xml
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]:
```suggestion
# object_store_config_file = config/object_store_conf.xml
```

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 5]
```diff
@@ -0,0 +1,1434 @@
...
+# 'filter-with' in the [app:main] section below.
+
+# Define the gzip filter.
+[filter:gzip]
+use = egg:Paste#gzip
+
+# Define the proxy-prefix filter.
+[filter:proxy-prefix]
+use = egg:PasteDeploy#prefix
+prefix = /galaxy
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]:
```suggestion
# prefix = /galaxy
```

File: lib/galaxy/config/sample/galaxy_vscode.ini [hunk 6]
```diff
@@ -0,0 +1,1434 @@
...
+# Configuration of the internal HTTP server.
+
+[server:main]
+
+# The internal HTTP server to use.  Currently only Paste is provided.  This
+# option is required.
+use = egg:Paste#http
+
+# The port on which to listen.
+#port = 8080
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/galaxy_vscode.ini]:
```suggestion
port = 8080
```

**Comment by kxk302** [comment]
Removed the commented out lines in the ini file. Now it's compact. 
@dannon would you mind looking at this change? (I think its too late for Marius now)

**Comment by kxk302** [comment]
All the checks are green. @mvdbeek or @dannon, would one of you mind merging this? Thx

**Comment by kxk302** [comment]
Thx @dannon!

**Pull request merged**